aggressive-fill-paragraph.el

```diff
@@ -59,11 +59,20 @@ by *, + or -."
        (or (eql (org-element-type (org-element-at-point)) 'table)
            (eql (org-element-type (org-element-at-point)) 'table-row))))
 
+(defun afp-in-org-src-block-header? ()
+  (interactive)
```

Comment:
This probably shouldn't be interactive otherwise it will show up for people running commands.